BrickKit/app/src/main/java/com/wayfair/brickkitdemo/mainfragment/MainActivityFragment.kt

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright © 2017-2021 Wayfair. All rights reserved.
+ */
+package com.wayfair.brickkitdemo.mainfragment
+
+import android.os.Bundle
+import android.os.Handler
+import android.os.Looper
+import androidx.fragment.app.Fragment
+import com.wayfair.brickkit.brick.BaseBrick
+import com.wayfair.brickkit.size.BrickSize
+import com.wayfair.brickkit.size.PercentageBrickSize
+import com.wayfair.brickkitdemo.AddRemoveBrickFragment
+import com.wayfair.brickkitdemo.BrickFragment
+import com.wayfair.brickkitdemo.FragmentBrickFragment
+import com.wayfair.brickkitdemo.InfiniteScrollBrickFragment
+import com.wayfair.brickkitdemo.R
+import com.wayfair.brickkitdemo.SimpleBrickFragment
+import com.wayfair.brickkitdemo.StaggeredInfiniteScrollBrickFragment
+import com.wayfair.brickkitdemo.mainfragment.bricks.PassiveBrick
+import com.wayfair.brickkitdemo.mainfragment.bricks.ActiveBrick
+
+/**
+ * Main fragment that links to the other child fragments. Additionally this shows an example of placeholders.
+ */
+class MainActivityFragment : BrickFragment() {
+    private var rowNumber = 0
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        val placeholderBrick = ActiveBrick(resources, R.string.empty) { changeFragment(SimpleBrickFragment()) }
+            .also {
+                // Fake waiting as in a API request to show off placeholder when data is not ready
+                Handler(Looper.getMainLooper()).postDelayed(
+                    { it.text = getString(R.string.simple_brick_title) },
+                    2000
```

Comment:
Should use something like TimeUnit.SECONDS.toMillis(2) I think?

Reply:
done